ackrm:
  # Publish all joint states -----------------------------------
  joint_state_ctrl:
    type: joint_state_controller/JointStateController
    publish_rate: 30 

  # Wheels velocity controller ---------------------------------
  wheels_vel_ctrl:
    type: velocity_controllers/JointGroupVelocityController
    joints:
      - front_right_wheel_joint
      - front_left_wheel_joint
      - back_right_wheel_joint
      - back_left_wheel_joint

  # Steering controller for the left wheel ---------------------
  l_steer_ctrl:
    type: position_controllers/JointPositionController
    joint: left_steer_joint

  # Steering controller for the right wheel --------------------
  r_steer_ctrl:
    type: position_controllers/JointPositionController
    joint: right_steer_joint

  gazebo_ros_control:
    pid_gains:
      right_steer_joint: {p: 5.0, i: 0.05, d: 0.1}
      left_steer_joint: {p: 5.0, i: 0.05, d: 0.1}
      front_right_wheel_joint: {p: 10.0, i: 0.01, d: 0.0}
      front_left_wheel_joint: {p: 10.0, i: 0.01, d: 0.0}
      back_right_wheel_joint: {p: 10.0, i: 0.01, d: 0.0}
      back_left_wheel_joint: {p: 10.0, i: 0.01, d: 0.0}
